cloud/docker/docker_network.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+#
+# Copyright 2016 Red Hat | Ansible
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+module: docker_network
+version_added: "2.2"
+short_description: Manage Docker networks
+description:
+  - Create/remove Docker networks and connect containers to them.
+  - Performs largely the same function as the "docker network" CLI subcommand.
+options:
+  name:
+    description:
+      - Name of the network to operate on.
+    default: null
+    required: true
+    aliases:
+      - network_name
+
+  connected:
+    description:
+      - List of container names or container IDs to connect to a network.
+    default: null
+
+  driver:
+    description:
+      - Specify the type of network. Docker provides bridge and overlay drivers, but 3rd party drivers can also be used.
+    default: bridge
+
+  driver_options:
+    description:
+      - Dictionary of network settings. Consult docker docs for valid options and values.
+    default: null
+
+  force:
+    description:
+      - >
+        With state 'absent' forces disconnecting all containers from the
+        network prior to deleting the network. With state 'present' will
+        disconnect all containers, delete the network and re-create the
+        network.  This option is required if you have changed the ipam or
+        driver options and want an existing network to be updated to use the
+        new options.
+    default: false
+
+  incremental:
```

Comment:
make this `appends` to keep in line with other modules, incremental can still be an alias